rclcpp/include/rclcpp/memory_strategy.hpp

```diff
@@ -101,6 +155,11 @@ class MemoryStrategy
   std::vector<rclcpp::subscription::SubscriptionBase::SharedPtr> subs;
   std::vector<rclcpp::service::ServiceBase::SharedPtr> services;
   std::vector<rclcpp::client::ClientBase::SharedPtr> clients;
+
+  std::vector<void *> subscription_handles;
```

Comment:
Is there a max number of handles supported which could be reserved in the constructor? Using resize may or may not allocate unless it's been previously allocated.